tests/job.go

```diff
@@ -134,16 +132,16 @@ else
   echo "failed"
   exit 1
 fi`,
-		strings.Join(checkConnectivityCmdPrefixes, ";"), timeout, host, port)
+		timeout, host, port)
 
 	return newHelloWorldJob(check)
 }
 
 // NewHelloWorldJobHTTP gets an IP address and a port, which it uses to create a pod.
 // This pod tries to contact the host on the provided port, over HTTP.
 // On success - it expects to receive "Hello World!".
-func NewHelloWorldJobHTTP(host string, port string, checkConnectivityCmdPrefixes ...string) *batchv1.Job {
-	check := fmt.Sprintf(`set -x; %sx="$(head -n 1 < <(curl --http0.9 %s:%s))"; echo "$x" ; if [ "$x" = "Hello World!" ]; then echo "succeeded"; exit 0; else echo "failed"; exit 1; fi`, strings.Join(checkConnectivityCmdPrefixes, ";"), FormatIPForURL(host), port)
```

Comment:
This is a utility method that allows for a set of prefixes to be optionally added. You're not forced to use it - in fact, you can pretty much have this change without changing the signature to this method.

Reply:
No one uses it and it complicates this helper. What reasoning you have to leave it?

Reply:
That it is useful, and optional. 

As I've said, this is opinionated. If you want it gone, delete it, I won't complain further.

Reply:
This is odd. You say it is opinionated, but you nack-ed it. I do not want to re-ack it without making sure you are ok with it.

Reply:
It's OK, you can lgtm it once again; I didn't know it would remove the lgtm tag (should have commented instead of requesting changes).

Sorry for the extra step.

Reply:
[Here](https://github.com/kubevirt/kubevirt/pull/5200#discussion_r619830610) for example I suggested Radim to add another varargs to the method.
Keeping the existing (non used) varargs will complicate the implementation for Radim.

Reply:
> [Here](https://github.com/kubevirt/kubevirt/pull/5200#discussion_r619830610) for example I suggested Radim to add another varargs to the method.
> Keeping the existing (non used) varargs will complicate the implementation for Radim.

First I heard of that. I don't think I had a fair chance of knowing that was part of your plans since it wasn't mentioned anywhere. That PR was not referenced here, afaict. I apologize if it was and I failed to notice it.

Reply:
It wasn't mentioned here. I believe this PR stands as it's own. This was just an example of why keeping non used args may complicate things in the future.

I think we all agree an got the point:)